virtualizarr/tests/test_writers/test_icechunk.py

```diff
@@ -268,7 +273,10 @@ def test_write_loadable_variable(
     )
     vds = Dataset({"air": la_v}, {"pres": ma_v})
 
-    dataset_to_icechunk(vds, icechunk_filestore)
+    # Icechunk checksums currently store with second precision, so we need to make sure
+    # the checksum_date is at least one second in the future
+    checksum_date = datetime.now(timezone.utc) + timedelta(seconds=1)
+    dataset_to_icechunk(vds, icechunk_filestore, last_updated_at=checksum_date)
```

Comment:
We don't appear to have a test that defines the expected behaviour for the default value. It might also be good to raise an error if someone passes a value in the future (if icechunk doesn't do that already).

Reply:
yeah icechunk tests this very comprehensively, but i figured it would be good to have at least one test that threads the values through